dozer/cogs/levels.py

```diff
@@ -311,6 +334,7 @@ async def configureranks(self, ctx):
             embed.add_field(name=f"Levels are {enabled} for {ctx.guild}", value=f"XP min: {settings.xp_min}\n"
                                                                                 f"XP max: {settings.xp_max}\n"
                                                                                 f"Cooldown: {settings.xp_cooldown} Seconds\n"
+                                                                                f"Keep old roles: {settings.keep_old_roles}\n"
```

Comment:
This should be added to the example usage as well. (This comment would be on line 343 if GitHub allowed commenting on unchanged lines.)